open-machine-learning-jupyter-book/prerequisites/python-programming-advanced.md

```diff
@@ -1116,7 +1007,7 @@ Now what can we do with instance objects? The only operations understood by inst
 - https://docs.python.org/3/tutorial/classes.html#instance-objects
 ```
 
-DATA ATTRIBUTES need not be declared; like local variables, they spring into existence when they are first assigned to. For example, if x is the instance of MyCounter created above, the following piece of code will print the value 16, without leaving a trace.
+DATA ATTRIBUTES need not be declared; like local variables, they spring into existence when they are first assigned to. For example:

```

Comment:
why the second half is removed?

Reply:
The second half of the sentence has nothing to do with the context。